lib/datasource/local/settings_storage.dart

```diff
@@ -168,27 +168,22 @@ class _SettingsStorage {
 
   void enableRecoveryMode({required String accountName, String? privateKey}) {
     inRecoveryMode = true;
-    this.accountName = accountName;
+    _accountName = accountName;
     this.privateKey = privateKey;
   }
 
   void finishRecoveryProcess() => inRecoveryMode = false;
 
   void cancelRecoveryProcess() {
     inRecoveryMode = false;
-    accountName = null;
+    _accountName = null;
     privateKey = null;
   }
 
   void savePasscode(String? passcode) => this.passcode = passcode;
 
   void saveAccount(String accountName, String privateKey) {
-    this.accountName = accountName;
-    this.privateKey = privateKey;
-  }
-
-  void saveAccountFromWaitingForRecover(String accountName, String privateKey) {
```

Comment:
@RaulUrtecho I removed this method again since it's the same as saveAccount, and it's also redundant - see other comment

Reply:
Agree